### PR TITLE
Type definitions for ip-regex v3

### PR DESCRIPTION
**Pull request by renarsvilnis**
Hi, updated the type definitions for ip-regex v3. Just starting out with typescript so a code review would be great.

Because of the design of the module had add:

`// tslint:disable-next-line no-single-declare-module no-declare-current-package`

line to the module declaration. Not sure if its okay to disable it, saw node type definitions using it.

----

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/sindresorhus/ip-regex
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/ip-regex/index.d.ts [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+// Definitions by: Renārs Vilnis <https://github.com/renarsvilnis>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/** Regular expression for matching IP addresses (IPv4 & IPv6). */
-declare function ipRegex(options?: {
-    /** Only match an exact string. Useful with RegExp#test to check if a string is an IP address. */
-    exact?: boolean;
-}): RegExp;
+// tslint:disable-next-line no-single-declare-module no-declare-current-package
+declare module "ip-regex" {
```
Comment (sandersn) [inline types/ip-regex/index.d.ts]: the declare module wrapper is not needed here.

File: types/ip-regex/index.d.ts [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+    function ipRegex(options?: ipRegex.IpRegexOptions): RegExp;
```
Comment (sandersn) [inline types/ip-regex/index.d.ts]: the source calls this function `ip`. It doesn't matter if the name is different, but it's more confusing to me.

File: types/ip-regex/ip-regex-tests.ts [hunk 3]
```diff
@@ -1,7 +1,13 @@
-import ipRegex = require('ip-regex');
+import * as ipRegex from 'ip-regex';
```
Comment (sandersn) [inline types/ip-regex/ip-regex-tests.ts]: the sources uses commonjs exports, so `export = ipRegex` and `import ipRegex = require('ip-regex')` is correct. Please revert this line.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/ip-regex/index.d.ts (hunk 2 above)
Comment (renarsvilnis) [reply types/ip-regex/index.d.ts]: Fixed

**Review by sandersn — APPROVED** [review]

**Pull request merged**